src/utils/prefixedAddress.ts

```diff
@@ -0,0 +1,13 @@
+import { getCurrentShortChainName } from 'src/config'
+import { SHORT_NAME } from 'src/config/networks/network.d'
+
+export const isValidPrefix = (prefix: string): boolean => {
+  return Object.values(SHORT_NAME).includes(prefix as SHORT_NAME)
+}
+
+export const parsePrefixedAddress = (fullAddress: string): { address: string; prefix: string } => {
+  const parts = fullAddress.split(':').filter(Boolean)
```

Comment:
Wow, what is the `filter(Boolean)` ? This code is neat!

Reply:
Filters empty strings. For inputs like `rin:` w/o any address.